python/dlr/api.py

```diff
@@ -71,7 +72,7 @@ def _check_call(ret):
     if ret != 0:
         raise DLRError(_LIB.DLRGetLastError().decode('ascii'))
 
-class DLRModel:
+class DLRModelImpl:
```

Comment:
`DLRModel` dispatcher does not need `ctypes`, `numpy` and `_LIB = _load_lib()` in case we are working with `.pb` file.
I think `api.py` should contain minimum imports/code. It only needs the interface and the dispatcher.
I'd move `DLRModelImpl` and all its imports to `dlr_model.py` file (module).

Reply:
Hi Alex, we can always put imports right before where they're used, so I think the real question is do we make dlr_model and tf_model equal implementations (your suggestion), or do we treat tf_model as an extension/bypass to dlr (current structure), since tf_model relies on external python pkgs. 

Thoughts?

Reply:
I think that from architectural point of view DLRModelImpl is equal to TFModelImpl.